apiary.apib

```diff
@@ -1893,6 +1893,7 @@ Get details for a single invoice.
                                 + type: `productCategory` (string)
                             + quantity: 3 (number)
                             + description: `An awesome product` (string)
+                            + extended_description: `Some more information about this awesome product` (string)
```

Comment:
Q: is this an html field in the application? Otherwise it might be useful to note that it has to be send as markdown?

Reply:
I don't think so, but I'll verify.

Reply:
Hmm, apparently it is. Should we add that to the documentation here? Do we do that in any other places?

Reply:
I would add it as a comment. Not sure if this is already documented for other fields or if there is a datatype for it. I'll check.

Reply:
So, apparently it is not documented anywhere 🤦‍♂️

Reply:
I'll make sure this gets documented for the core endpoints.

Reply:
Maybe not that important for the read side though, but should especially be specified for write endpoints imo.